wayback-core/src/main/java/org/archive/wayback/util/webapp/RequestMapper.java

```diff
@@ -166,6 +166,15 @@ public void addRequestHandler(int port, String host, String path,
 			portMap.put(key, portMapper);
 		}
 		portMapper.addRequestHandler(path, requestHandler);
+		// path-segment parameter support
```

Comment:
This appears to register the path requestHandler twice. Would it be better to wrap this and line 168 above in an `if...else` block?

Reply:
Is that about line 168 and 174? Probably comment 169--172 is not clear, but  I need to register twice, for example, one for "/save;" and another for "/save", so that the request without segment parameters are routed correctly.